webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePrimaryActionButton.js

```diff
@@ -23,14 +23,19 @@ const PrimaryActionButton = ({ action }) => {
 
 // eslint-disable-next-line react/prop-types
 const urlButton = ({ url, title }) => (
-  <Button href={url} bsStyle="primary" bsSize="large">
+  <Button component="a" href={url} target="_blank" variant="primary">
     {title}
   </Button>
 );
 
 // eslint-disable-next-line react/prop-types
 const onClickButton = ({ onClick, title }) => (
-  <Button onClick={onClick} bsStyle="primary" bsSize="large">
+  <Button
+    component="a"
+    inoperableEvents={onClick}
```

Comment:
I am not sure how it works, their docs are not very informative about it
I would search how other consumers used the `onClick`

Reply:
Yeah, need to look into the `onClick` part!

Reply:
According to the PF4's [implementation](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/Button/Button.tsx) the default rendered `component` is basically an HTML button element, plus the `<Button>` component passed all given extra props, so this should work:
```js
<Button onclick={onClick} variant='primary'> {title} </Button>
```